langs/java.go

```diff
@@ -206,9 +206,8 @@ func (lh *JavaLangHelper) getFDKAPIVersion() (string, error) {
 	}
 
 	// nishalad95: bin tray TLS certs cause verification issues on OSX, skip TLS verification
-	transCfg := &http.Transport{
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: true}, // ignore expired SSL certificates
-	}
+	transCfg := http.DefaultTransport
```

Comment:
`http.DefaultTransport` is an interface so this will modify by reference( modifying this will modify the global behaviour ) 
it's messy but it's better to assign a copy: 
```
	var transCfg http.Transport
	transCfg = *(http.DefaultTransport.(*http.Transport))
	transCfg.TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
	client := &http.Client{Transport: &transCfg}

```

Reply:
This will result in copying the `idleMu     sync.Mutex` from DefaultTransport which I think is not a good idea? Copying in values from DefaultTransport seems the way to go.